### ISSUE TITLE
[jsdoc/require-jsdoc] : 'CallExpression:not(*)' == errors triggered on functions calls

### ISSUE THREAD

**Issue by RichardFevrier** [gajus/eslint-plugin-jsdoc#614]
### Expected behavior

When using the rule `jsdoc/require-jsdoc` with `contexts` with `CallExpression:not(*)` and `MemberExpression`, the plugin should not trigger an error on functions calls.

### Actual behavior
Errors are triggered on functions calls.

### ESLint config

```js
module.exports = {
  root: true,
  globals: {
    'console': false
  },
  parser: 'babel-eslint',
  extends: ['plugin:jsdoc/recommended'],
  rules: {
    'jsdoc/require-jsdoc': ['error', {
      contexts: [
        'CallExpression:not(*)', // should prevent the next line (also tried `'CallExpression:not(*) > MemberExpression'` but wasn't sure about this one)
        'MemberExpression', // <- false positive
        'VariableDeclaration',
      ],
    }],
  },
};
```

### ESLint sample

```js
class Test {

  constructor() {
    this.memberExpression = 'member'; // good
    this.aFunc(); // wrong: should not be triggered
    const aVariable = 'variable'; // good
  }

  /**
   * @returns {void}
   */
  aFunc = () => {}
}
```

### CLI sample
```eslint myfile.js```

### Environment

- Node version: 12.16.3
- ESLint version: 6.8.0
- eslint-plugin-jsdoc version: 30.0.3

**Comment by brettz9** [comment]
It seems you want: `*:not(CallExpression) > MemberExpression` (and remove `CallExpression:not(*)` and `MemberExpression`.

**Comment by RichardFevrier** [comment]
Right @brettz9 thank you !
I'm sorry you can close this issue.
